### ISSUE TITLE
templates.py: future-proof

### ISSUE THREAD

**Issue by Andrei-Pozolotin** [ostr00000/freecad-stubs#10]
after upgrading to the latest `stubs`:
https://pypi.org/project/freecad-stubs/1.0.16/

and building to the latest `FreeCAD`:
```
OS: Arch Linux (KDE/plasma)
Word size of FreeCAD: 64-bit
Version: 0.22.0dev.35325 (Git)
Build type: None
Branch: makepkg
Hash: 5c2eedbd149536f88cf70b085d7afff101fa2037
Python 3.11.6, Qt 5.15.11, Coin 4.0.1, Vtk 9.3.0, OCC 7.7.2
```

and copying `templates.py` "almost-as-is" into the plugin-under-development,

there is an import error on startup:
```
During initialization the error "module 'FreeCADGui' has no attribute 'View3DInventorViewerPy'" occurred ...
```

without bothering about "why and where", one way to future-proof against this is to include in `templates.py`:
```python
from __future__ import annotations
```

**Comment by ostr00000** [comment]
There may be the following problem:
some C++ classes have Python proxy, but are not available to import.

This is a problem in runtime. I placed these classes in the stubs, so it should be available only for typing.

Adding `from __future__ import annotations` is the correct solution, so I included it in [code](https://github.com/ostr00000/freecad-stubs/commit/5dcafad26a620cf1265abbddbe7436410dd560d0).
